composer.lock

```diff
@@ -0,0 +1,3851 @@
+{
```

Comment:
While it may sound strange to have a `composer.lock` in a library, beware that:

 1.  it is not considered by downstream consumers
 2. it is here mostly to tell you if future failures are caused by a change in dependencies, or in your code (otherwise, you may get random failures if, for example, phpunit changes slightly)